ui/apps/platform/src/Containers/VulnMgmt/Entity/Policy/VulnMgmtPolicyOverview.js

```diff
@@ -122,20 +134,6 @@ const VulnMgmtPolicyOverview = ({ data, entityContext, setRefreshTrigger }) => {
         </div>
     );
 
-    // @TODO: extract this out to make it re-usable and easier to test
```

Comment:
I'm guessing that this logic was put in before the `"Policy Violated"` search option was implemented, and was just never removed when the "# failing deployments" link was implemented. As far as I can tell this client side filtering achieves the same thing the new (circa 2020 "new") option does.